cmake/Modules/Findpq.cmake

```diff
@@ -40,9 +40,14 @@ if (NOT pq_FOUND)
   externalproject_add(postgres_postgres
       GIT_REPOSITORY  ${URL}
       GIT_TAG         ${VERSION}
-      CONFIGURE_COMMAND ./configure --without-readline
+      CONFIGURE_COMMAND
+                      ./configure --without-readline
+                      CC=${CMAKE_C_COMPILER}
+                      CXX=${CMAKE_CXX_COMPILER}
```

Comment:
`CXX` is probably unused https://www.postgresql.org/docs/9.5/install-procedure.html